collectors/idlejitter.plugin/README.md

```diff
@@ -5,16 +5,23 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/idlej
 
 # idlejitter.plugin
 
-It works like this:
+Idle jitter is the delay of a CPU process.
```

Comment:
Not an expert, my understanding of this plugin - it tries to measure CPU preemption latency (task scheduling). @netdata/agent, back me up on this.

Reply:
Both are kind of correct.

The internal measurement is making a call to sleep (really nanosleep or uslep, depending on the system), and then tracking how much longer it takes for the process to be woken up than the time it asked to sleep for. This is functionally ‘lost’ time from the  perspective of the process in question, and high values cause all kinds of problems for realtime and near-realtime applications.

It’s technically a function of preemption latency, and it is also a measure of (one type of) induced delay in processes utilizing such timing functions

I would suggest phrasing it as:

```suggestion
Idle jitter is a measure of delays in timing for user processes caused by scheduling limitations.
```